src/style.css

```diff
@@ -1,3 +1,79 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  width: 1200px;
+}
+
+header {
```

Comment:
Avoid using tag names for styling, except for `html` and `body`. Consider using a class for the header styling.